<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCommon">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Data directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="leDataDir">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>./data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="tbSelDataDir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Query text directoty</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="leQueryDir">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>./query</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="tbSelQueryDir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbLanguage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QCheckBox" name="chShowBtd">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>7</y>
            <width>20</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lbShowBtd">
          <property name="geometry">
           <rect>
            <x>29</x>
            <y>2</y>
            <width>300</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>Show buttons to edit the table &quot;Total day data&quot;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDailyRate">
      <property name="toolTip">
       <string>These ratios are used when creating a new file</string>
      </property>
      <attribute name="title">
       <string>Percent ratio</string>
      </attribute>
      <widget class="QTableWidget" name="twRatio">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>25</y>
         <width>326</width>
         <height>120</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>72</number>
       </attribute>
       <row>
        <property name="text">
         <string>Proteins</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Fats</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Carbohydrates</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Slimming</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Norm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Muscle gain</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string notr="true">20</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string notr="true">30</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string notr="true">30</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string notr="true">20</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string notr="true">30</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string notr="true">20</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string notr="true">60</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string notr="true">40</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string notr="true">50</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>5</y>
         <width>326</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Ratio of proteins, fats, carbohydrates for:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSetDefaultRatio">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>174</y>
         <width>170</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set default ratio</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>154</y>
         <width>73</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Variation</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sbVariation">
       <property name="geometry">
        <rect>
         <x>85</x>
         <y>150</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string>± </string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>2.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>154</y>
         <width>70</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>percents</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
